Localization/de-DE/statistics.yaml

```
@@ -0,0 +1,45 @@
+---
+url: "https://foldingathome.org/statistics/"
+language: "de-DE"
+title: "STATISTIK"
+text: [
+# First two paragraphs
+#"One of the best ways to help Folding@home is by recruiting your friends and family. Start by sharing our project with them. Then join a team or even start your own team. The more points your team earns, the closer we come to finding cures.
+#On this page you will find access to statistics for individuals and teams who have joined together to earn points and compete with other teams. Some of us are quite intense in our approach to folding. We have team websites, we supe up our computers, and we drive the technology forward by reporting bugs and making suggestions about how to improve the software.",
+"Eine der besten Möglichkeiten, Folding@home zu helfen, besteht darin, Ihre Freunde und Familie zu rekrutieren. Teilen Sie zunächst unser Projekt mit ihnen. Schließen Sie sich dann einem Team an oder gründen Sie sogar ein eigenes Team. Je mehr Punkte Ihr Team verdient, desto näher kommen wir uns Heilmittel finden.
```

```suggestion
"Eine der besten Möglichkeiten, Folding@home zu helfen, besteht darin, Ihre Freunde und Familie zu rekrutieren. Teilen Sie zunächst unser Projekt mit ihnen. Schließen Sie sich dann einem Team an oder gründen Sie sogar ein eigenes Team. Je mehr Punkte Ihr Team verdient, desto näher kommen wir einem Heilmittel.
```
Small correction for the end of that sentence.